heaps/min_heap.py

```diff
@@ -19,18 +19,29 @@ def __init__(self):
     def add(self, key, value = None):
```

Comment:
👀 Nice work, however space complexity would be O( log n) because of the recursive calls in `heap_up`